mocks/mock-data/disputes.ts

```diff
@@ -48,6 +48,207 @@ const VISA_CONSUMER_DEFENSE_REASONS = ['InvalidChargeback', 'MerchandiseReceived
+    isRefunded: false,
+    merchantReference: 'b2c3d4e5-f6g7-5890-efgh',
+    paymentMethod: { lastFourDigits: '1234', type: 'visa' },
+    pspReference: 'a1b2c3d4-e5f6-4789-abcd',
+};
+
+const DEFAULT_DETAIL_DISPUTE: IDisputeDetail['dispute'] = {
+    amount: {
+        value: 211100,
+        currency: 'EUR',
+    },
+    dueDate: new Date(new Date().setHours(23)).toISOString(),
+    createdAt: getDate(-10),
+    pspReference: 'a1b2c3d4-e5f6-4789-abcd-000000000001',
+    reason: {
+        category: 'REQUEST_FOR_INFORMATION',
+        title: 'Request for information',
+        code: '1234',
+    },
+    allowedDefenseReasons: [],
+    defensibility: 'ACCEPTABLE',
+    type: 'CHARGEBACK',
+    status: 'UNDEFENDED',
+};
+
+const DEFAULT_DISPUTE_DETAIL: IDisputeDetail = {
+    dispute: DEFAULT_DETAIL_DISPUTE,
+    payment: DEFAULT_DETAIL_PAYMENT,
+};
+
+// CHARGEBACKS
+
+export const CHARGEBACK_UNDEFENDED: IDisputeDetail = {
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'UNDEFENDED',
+    },
+    payment: DEFAULT_DETAIL_PAYMENT,
+};
+
+export const CHARGEBACK_WON: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'WON',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+    defense: DEFAULT_DETAIL_DEFENSE,
+};
+
+export const CHARGEBACK_LOST: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'LOST',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+    defense: DEFAULT_DETAIL_DEFENSE,
+};
+
+export const CHARGEBACK_ACCEPTED: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'ACCEPTED',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+};
+
+export const CHARGEBACK_PENDING_DEFENDABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    payment: {
+        ...DEFAULT_DETAIL_PAYMENT,
+        paymentMethod: {
+            type: 'mc',
+            lastFourDigits: '4509',
+        },
+    },
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        reason: {
+            category: 'CONSUMER_DISPUTE',
+            code: 'AirlineFlightProvided',
+            title: 'Consumer dispute',
+        },
+        status: 'UNDEFENDED',
+        defensibility: 'DEFENDABLE',
+        allowedDefenseReasons: [...MC_CONSUMER_DEFENSE_REASONS],
+    },
+};
+export const CHARGEBACK_PENDING_DEFENDABLE_EXTERNALLY: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'UNDEFENDED',
+        defensibility: 'DEFENDABLE_EXTERNALLY',
+    },
+};
+export const CHARGEBACK_PENDING_ACCEPTABLE: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DETAIL_DISPUTE,
+        status: 'UNDEFENDED',
+        defensibility: 'ACCEPTABLE',
+    },
+};
```

Comment:
Consider renaming this to just `CHARGEBACK_ACCEPTABLE` to avoid the confusion with the `pending` word. Also, `defense` field should not be in the data. Consider this:

```suggestion
export const CHARGEBACK_ACCEPTABLE: IDisputeDetail = {
    dispute: {
        ...DEFAULT_DETAIL_DISPUTE,
        defensibility: 'ACCEPTABLE',
        status: 'UNDEFENDED',
    },
    payment: DEFAULT_DETAIL_PAYMENT,
};
```

Reply:
✅